conf/nginx.conf

```diff
@@ -1,7 +1,7 @@
 #sub_path_only rewrite ^__PATH__$ __PATH__/ permanent;
 location @api {
 
-  proxy_pass http://localhost:__PORT__;
```

Comment:
Are you sure it would work? I am pretty sure Grist expects to listen on a domain like localhost, and that we have tried this before and got bad gateways.